reference_docs/YAML_models/standoc_document_attributes.yaml

```diff
@@ -0,0 +1,724 @@
+    description: "Abbreviation of the subdivision of the organization that is responsible for this document".
+    added_in_version: v1.6.1
+
+  pub-address:
+    tags: ["Author info"]
+    title: "Address of the organization"
+    description: |
+      "Address of the organization responsible for this document, if it overrides
+      the default.
+      +
+      [NOTE]
+      --
+      Each line in a multi-line address must end with `+ \`, e.g.
+
+      [source,adoc]
+      ----
+      :pub-address: 1 Infinity Loop + \
+      California + \
+      United States of America
+      ----
+      --".
+    added_in_version: v1.6.1
+
+  pub-phone:
+    tags: ["Author info"]
+    title: "Phone number of the organization"
+    description: "Phone number of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+  pub-fax:
+    tags: ["Author info"]
+    title: "Fax number of the organization"
+    description: "Fax number of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+  pub-email:
+    tags: ["Author info"]
+    title: "Email of the organization"
+    description: "Email of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+  pub-uri:
+    tags: ["Author info"]
+    title: "URI of the organization"
+    description: "URI of the organization responsible for this document, if it overrides the default".
+    added_in_version: v1.6.1
+
+
+  #--- Visual appearance ---#
+  body-font:
+    tags: ["Visual appearance"]
+    title: "Body font"
+    description: "Font for body text; will be inserted into CSS, overriding the default set for the particular Metanorma flavour.".
+
+  header-font:
+    tags: ["Visual appearance"]
+    title: "Header font"
+    description: "Font for headers; will be inserted into CSS, overriding the default set for the particular Metanorma flavour.".
+
+  monospace-font:
+    tags: ["Visual appearance"]
+    title: "Monospace font"
+    description: "Font for monospace; will be inserted into CSS, overriding the default set for the particular Metanorma flavour.".
+
+  htmlstylesheet:
+    tags: ["Visual appearance"]
+    title: "HTML stylesheet"
+    description: "SCSS stylesheet to use for HTML output. Defaults to built-in template for the particular Metanorma flavour. Overriding is not recommended.".
+
+  htmlstylesheet-override:
+    tags: ["Visual appearance"]
+    title: "HTMLstylesheet override"
+    description: "CSS stylesheet to use for HTML output. Is inserted after any built in stylesheet for the particular Metanorma flavour, and can be used to override it.".
+    added_in_version: v1.8.7
+
+  htmlcoverpage:
+    tags: ["Visual appearance"]
+    title: "HTML cover page"
+    description: |
+      "HTML template for cover page.\n
+      Defaults to built-in template for the particular Metanorma flavour.\n
+      Overriding is not recommended.".
```

Comment:
The template for the coverpage involves knowing not just how the CSS works, but how the metadata template populating HTML works: https://www.metanorma.org/builder/topics/metadata-and-boilerplate/ (and we do end up tinkering with Metadata a fair bit without necessarily documenting it, because we haven't been assuming external use.) There is also branding and information compliance at issue.